cmd/mdatagen/main.go

```diff
@@ -104,7 +99,7 @@ func generateFile(tmplFile string, outputFile string, md metadata) error {
 				},
 				"stringsJoin": strings.Join,
 				"inc":         func(i int) int { return i + 1 },
-			}).ParseFiles(tmplFile))
+			}).ParseFS(templateFS, strings.ReplaceAll(tmplFile, "\\", "/")))
```

Comment:
So looking at the coder, it looks like the `\` is a special character used for `template.Glob` and caused issues there.

Reply:
Please add a comment reflecting the reason for this `ReplaceAll` here.  Is this a known issue w/ ParseFS?

Reply:
Added.

I haven't checked if there is a known issue with `ParseFS` for windows

Reply:
From what I understand, it looks like it extends from https://pkg.go.dev/io/fs#ValidPath which forces `/` as the path separator regardless of OS.